Kernel/Bus/USB/USBDevice.cpp

```diff
@@ -97,4 +97,37 @@ ErrorOr<size_t> Device::control_transfer(u8 request_type, u8 request, u16 value,
     return TRY(m_default_pipe->submit_control_transfer(request_type, request, value, index, length, data));
 }
 
+ErrorOr<void> Device::activate_configuration(USBConfiguration const& configuration)
+{
+    if (m_was_configured.was_set() && m_current_configuration != configuration.configuration_id())
```

Comment:
Tracking interface usage is a bit harder to track with minimal invasiveness, but might be missing something
I can remove the check but prefer to be better safe now then forgetting it later

Reply:
Sure, let's keep that check for now then. We can maybe remove it once we have a better way of probing USB drivers instead of trying all drivers one by one. That theoretical new abstraction would probably have to keep track of configurations itself.